rpm/update-config-files.sh

```diff
@@ -54,11 +48,8 @@ if [[ -n "$FEDORA" ]] || [[ "$RHEL" -ge 10 ]]; then
     sed -i -e '/^additionalimagestores\ =\ \[/a "\/usr\/lib\/containers\/storage",' storage.conf
 fi
 
-# Set these on Fedora Rawhide (41+) and RHEL 10+
+# Set these on Fedora 41+ and RHEL 10+
 # regardless of distro
 if [[ "$FEDORA" -gt 40 ]] || [[ "$RHEL" -ge 10 ]]; then
     ensure pkg/config/containers.conf   compression_format  \"zstd:chunked\"
-    ensure storage.conf pull_options    \{enable_partial_images\ =\ \"true\",\ use_hard_links\ =\ \"false\",\ ostree_repos=\"\",\ convert_images\ =\ \"false\"\}
-    # Leave composefs disabled
-    ensure storage.conf use_composefs   \"false\"
```

Comment:
These things just go away? Is that intentional?

Reply:
Bump?

Reply:
They are all c/storage defaults AFAICT so why should they be set in the file?

Reply:
For the record, `enable_partial_images` is on by default since https://github.com/containers/storage/pull/1833 .

Reply:
I think these comments were posted after the merge. This change didn't make it in but anyway.

IIUC, @jnovy's intention behind `ensure`  was to actually ensure/test that those options are set as expected, regardless of upstream defaults, correct me if I'm wrong Jindrich. I think this ensure functionality can be moved to TMT tests which run on rpms installed from copr jobs on PRs.